dockers/docker-database/database_config.json

```diff
@@ -3,7 +3,8 @@
         "redis":{
             "hostname" : "127.0.0.1",
             "port" : 6379,
-            "unix_socket_path" : "/var/run/redis/redis.sock"
+            "unix_socket_path" : "/var/run/redis/redis.sock",
+            "persistence" : "yes"
```

Comment:
since it is not persistent for cold boot, should we make the name more specific? like persistence_for_warm_boot?